### PR TITLE
chore: added Harsh to projects #1652

### PR DESCRIPTION
**Pull request by HarshSharma0801**
#1652 
Added Project

### CODE REVIEW

**Comment by HarshSharma0801** [comment]
@priyankarpal

File: DB/projects.json [hunk 1]
```diff
@@ -12,6 +12,22 @@
...
+      "gitHub": "https://github.com/HarshSharma0801",
+      "LinkedIn": "https://www.linkedin.com/in/harsh-sharma-18a72b26a/"
+    
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/HarshSharma0801/SignUp-UsingNodejs",
+        "title": "SignUp Form Nodejs",
+        "description": "This is a database SignUp form for admin ",
+        "tech": ["html", "css", "javascript", "Nodejs" , "Expressjs" , "MailchimpApi" ]
```
Comment (priyankarpal) [inline DB/projects.json:28]: convert to lowercase 
```suggestion
        "tech": ["html", "css", "javascript", "nodejs" , "expressjs" , "mailchimpapi" ]
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by HarshSharma0801** [comment]
@priyankarpal done lowercase

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Pull request merged**